README.md

```diff
@@ -34,14 +34,27 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+The difference between the two is that responsive web design the layout its fluid the page resizes its width and height to adapt to different screen sizes for this reason its takes a lot longer to create and adaptive website is less flexible and needs design for each screen size to be created individually. 
+
```

Comment:
Also, adaptive designs take place at certain defined breakpoints with responsive design is implemented at any size.